packages/yourloops/lib/medical-files/medical-files.api.ts

```diff
@@ -25,29 +25,9 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 import HttpService from '../http/http.service'
-import { type Prescription } from './models/prescription.model'
 import { type MedicalReport, type NewMedicalReport } from './models/medical-report.model'
 
 export default class MedicalFilesApi {
-  static async uploadPrescription(teamId: string, patientId: string, prescriptorId: string, initialPeriod: number, file: File): Promise<Prescription> {
-    const formData = new FormData()
-    formData.append('patientId', patientId)
-    formData.append('prescriptorId', prescriptorId)
-    formData.append('initialPeriod', initialPeriod.toString())
-    formData.append('teamId', teamId)
-    formData.append('upload', file)
-    const { data } = await HttpService.post<Prescription, FormData>({
-      url: 'cargo/v0/prescriptions',
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Is this need to be removed from Cargo too ?

Reply:
It will have to eventually because it will be useless :slightly_smiling_face: